src/main/java/com/bobocode/hoverla/bring/context/BeanDefinitionValidator.java

```diff
@@ -151,6 +151,14 @@ private BeanDefinition resolveDependency(String dependencyName, Class<?> depende
             foundDependency = definitionByName.get();
             checkTypeMatching(dependencyType, foundDependency);
         } else {
+            // If the user provides a custom name for dependency - e.g. in annotation Qualifier, it must know about
+            // this dependency and is responsible for the dependency name. If the user didn't provide a name for
+            // the dependency, in this case, the name will be by default - class.getName(), so the finding process
+            // will be by the type too.
```

Comment:
I guess it's more about the case when we have wrong name in qualifier, innit?

Reply:
Yes. It's shorter and more correct. I'll change the comment.

Reply:
Done.